pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -597,6 +598,8 @@ func runHandler(ctx *domainContext, key string, c <-chan interface{}) {
 		case configArg, ok := <-c:
 			if ok {
 				config := cast.CastDomainConfig(configArg)
+				// Hardcoding for now
+				config.URL = "docker://zededa/zcli"
```

Comment:
Hey @spatchineelam I am not sure why this is needed, can you please explain? In fact I don't see it used for anything.

Reply:
These are draft code changes for Kalyan to pick up them and integrate with the downloaded changes he is making. Is there a way to flag the code as do not review....

Reply:
Sure, but why send a PR then if you don't want it to be reviewed @spatchineelam ? You can @kalyan-nidumolu can collaborate in your own repository on a branch and only send it for review when it is ready.